OmiseGOTests/APITests/RequestParametersTests.swift

```diff
@@ -1,47 +0,0 @@
-//
-//  RequestParametersTests.swift
-//  OmiseGOTests
-//
-//  Created by Mederic Petit on 21/3/18.
-//  Copyright © 2018 Omise Go Pte. Ltd. All rights reserved.
-//
-
-import XCTest
-@testable import OmiseGO
-
-class RequestParametersTests: XCTestCase {
```

Comment:
If we remove this file we should update tests in the other file if possible.
Note that these tests may appear obvious but it's more like a change-proof tests. Making sure that if we change something in the related code, these tests will fail and the developer will want to make sure that the changes were correct.